app/assets/javascripts/components/students/assign_button.jsx

```diff
@@ -123,7 +123,12 @@ const AssignButton = React.createClass({
       return;
     }
 
+    const closePopup = this.props.open;
+    const addAvailable = this.props.add_available;
```

Comment:
`addAvailable` is being used to determine whether to close the popup on confirmation. Naming it something like `closeOnConfirm` will make this a bit clearer, and will separate the role of the `onConfirm`, and will keep all the references to add_available together in the two setup lines.